themes/digital.gov/static/workflow/index.html

```diff
@@ -8,7 +8,7 @@
     <link rel="stylesheet" type="text/css" href="styles.css" />
```

Comment:
This script is used for netlify users to sign in to their account to access `workflow`.